src/controller.rs

```diff
@@ -153,6 +159,18 @@ impl Controller for FoxBox {
         format!("ws://{}:{}/services/{}/", self.hostname, self.ws_port, service_id)
     }
 
+    fn get_config(&self, namespace: String, property: String, default: String)
+        -> String {
```

Comment:
I just realized that the committed version got dumbed down from the original intention to also **set** the default value if it's not present. That allows efficient working with random values.

I went for option 2.